.NET/Microsoft.Recognizers.Text.DataDrivenTests/TestBase.cs

```diff
@@ -385,7 +385,7 @@ private void ValidateResults(TestModel testSpec, IEnumerable<string> testResolut
                         continue;
                     }
 
-                    Assert.AreEqual(expected.Resolution[key], actual.Resolution[key], GetMessage(testSpec));
+                    Assert.AreEqual(expected.Resolution[key].ToString(), actual.Resolution[key].ToString(), GetMessage(testSpec));

```

Comment:
The tests fail with the following message:
`Expected:<0.466666666666667>. Actual:<0.466666666666667>. Input: "أعتقد أن هذا سينجح. لذلك نعم سوف أقوم بذلك."`

I think the error here because of the delta issue in the double data type. so I changed them to string to match correctly.